README.md

```diff
@@ -1 +1,97 @@
-# Covid-19-app
\ No newline at end of file
+# CovidTracker--SARS-CoV-2.
+
+> This webapp provides Covid-19 Tracking information using the data it gets from an [API data source](https://corona.lmao.ninja) to provide comprehensive information about the novel coronavirus,
+>
+> It is created with [Create-React-App](https://github.com/facebook/create-react-app)
+
+---
+## Video Presentation
+(https://www.loom.com/share/8f32d2752b184da9ae05a6810dd2fd40)
+
```

Comment:
Please record your own presentation video and update the link with your recorded loom presentation video.